css/input.css

```diff
@@ -0,0 +1,39 @@
+.input_main {
```

Comment:
As I have seen in the above code you use "something-something", so you can use here also "-" instead of "_"  so it will be consistant.